src/client/components/vision/camera/model.ts

```diff
@@ -49,6 +51,50 @@ export class CameraModel {
   @observable.ref balls: Ball[]
   @observable.ref goals: Goal[]
   @observable.ref name: string
+  @observable draw = {
+    image: true,
+    compass: true,
+    horizon: true,
+    visualmesh: true,
+    balls: true,
+    goals: true,
+  }
+
+  @computed
+  get drawOptions() {
```

Comment:
Please move this computed prop to the component class, as this is not really part of the model.

i.e. models are just about data, they shouldn't have actions, or know specifics about the UI.